src/server/bootstrap.ts

```diff
@@ -132,6 +134,27 @@ export async function bootstrap() {
           res,
         };
       },
+      formatError: err => {
+        /**
+         * @see {@link https://www.apollographql.com/docs/apollo-server/data/errors/#for-client-responses}
+         */
+        // Don't give the specific errors to the client.
+
+        if (
+          err?.message?.includes(process.env.TYPEORM_DATABASE_HOST as string)
+        ) {
+          logger.error('DB connection error', err);
+          return new Error(errorMessages.ERROR_CONNECTING_DB);
```

Comment:
Is this error returned to the end-user? If yes it would be better to return internal server error and not make the user involved